tests/api/globalsearch.test.js

```diff
@@ -0,0 +1,70 @@
+// Copyright Contributors to the Open Cluster Management project
+
+jest.retryTimes(global.retry, { logErrorsBeforeRetry: true })
+
+const squad = require('../../config').get('squadName')
+const SEARCH_API_V1 = require('../../config').get('SEARCH_API_V1')
+const { getUserContext, getSearchApiRoute } = require('../common-lib/clusterAccess')
+const { ValidateSearchData, validationTimeout } = require('../common-lib/validateSearchData')
+const { resolveSearchItems } = require('../common-lib/searchClient')
+const { execSync } = require('child_process')
+const { execCliCmdString, expectCli } = require('../common-lib/cliClient')
+const { sleep } = require('../common-lib/sleep')
+
+const ns = 'search-rbac'
```

Comment:
Namespace should be unique to avoid collisions with the other test case.
```suggestion
const ns = 'search-global-rbac'
```

Reply:
This is what caused the deploy-and-run-e2e job to fail.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stolostron_search-e2e-test/351/pull-ci-stolostron-search-e2e-test-main-deploy-and-run-e2e/1703768567456993280#1:build-log.txt%3A163